### PR TITLE
chore: use notebooks routes

### PR DESCRIPTION
**Pull request by genehynson**
Use `/api/v2private/notebooks` routes instead of `/api/v2private/flows` routes

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 58 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=master)

File: package.json [hunk 1]
```diff
@@ -53,16 +53,15 @@
     "generate": "export SHA=f4424fc69ad29e31110d711310df644a57fee314 && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
     "generate-local": "export REMOTE=../openapi/ && yarn generate-meta",
     "generate-meta": "if [ -z \"${CLOUD_URL}\" ]; then yarn generate-meta-oss; else yarn generate-meta-cloud; fi",
-    "generate-meta-oss": "yarn oss-api && yarn notebooks && yarn unity && yarn flows && yarn managedFunctions && yarn annotations-oss",
-    "generate-meta-cloud": "yarn cloud-api && yarn notebooks && yarn unity && yarn flows && yarn managedFunctions && yarn annotations",
```
Comment (drdelambre) [inline package.json:57]: lol what a weird diff log

**Review by drdelambre — COMMENTED** [review]
does flowd still exist? curious about the migration pattern. like why isn't this behind a feature flag?

File: package.json (hunk 1 above)
Comment (genehynson) [reply package.json:57]: lol right?? Had to read through it a couple times to make sure the change actually makes sense 😆

**Comment by genehynson** [comment]
> does flowd still exist? curious about the migration pattern. like why isn't this behind a feature flag?

Flowd is dead. The /flows routes still exist behind notebooksd. I could do a feature flag...but it's so low risk I'm not sure it's worth it. In notebooksd, the /notebooks routes have an identical implementation to the /flows ones. Only the path is changing. But if you'd prefer, I can add a feature flag in

File: src/flows/context/api.tsx [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
-let stagedFlow: PatchFlowsOrgsFlowParams = DEFAULT_API_FLOW
+let stagedFlow: PatchNotebookParams = DEFAULT_API_FLOW
 let reportDecayTimeout = null
 let reportMaxTimeout = null
 
 const REPORT_DECAY = 500 // number of miliseconds to wait after last event before sending
 const REPORT_MAX_WAIT = 5000 // max number of miliseconds to wait between sends
 
-export const pooledUpdateAPI = (flow: PatchFlowsOrgsFlowParams) => {
+export const pooledUpdateAPI = (flow: PatchNotebookParams) => {
```
Comment (subirjolly) [inline src/flows/context/api.tsx:24]: NIT: If we are getting away from flow(s) then shouldn't we rename these vars and CONSTANTS to notebook(s)?

Comment (genehynson) [reply src/flows/context/api.tsx:24]: This is absolutely a fair point. But it's one of those things that if I start going down this train, there is soooooo much to refactor and rename 😭 I think it's an exercise we should do, but I think I'd like to do that in a separate PR.

Comment (drdelambre) [reply src/flows/context/api.tsx:24]: that was the same thinking that changed them all from notebooks to flows in the first place and broke things along the way. ROI seems... low. maybe i'm misunderstanding

**Review by drdelambre — APPROVED** [review]
if the implementations are exactly the same, then no need for a flag. just wanted to make sure there wasn't the chance of _also_ having a schema change or something hidden behind the change

**Comment by genehynson** [comment]
> if the implementations are exactly the same, then no need for a flag. just wanted to make sure there wasn't the chance of _also_ having a schema change or something hidden behind the change

In case anyone is curious:
- `/flows` routes: https://github.com/influxdata/idpe/blob/master/sc-frontend/notebooksd/src/routes/flows.ts
- `/notebooks` routes: https://github.com/influxdata/idpe/blob/master/sc-frontend/notebooksd/src/routes/notebooks.ts
- both use the `stream` controller: https://github.com/influxdata/idpe/blob/master/sc-frontend/notebooksd/src/controller/stream.ts

And fwiw I've had this change behind a flag for awhile now (`notebooksApi`) and have been testing it in prod - works fine :)

**Pull request merged**